### DISCUSSION TITLE
Remove ClientId from the path when uploading files to data lake

### DISCUSSION THREAD

**Discussion by germannolsen** [mindsphere/node-red-contrib-mindconnect#167] [Q&A] [answered]
When uploading files to data lake the root folder will always be named with the agent.ClientId(). We have a lot of devices sending files to date lake and we would like to create the folder structure by ourselves. Is there a workaround to decide the root folder name instead of using the  agent.ClientId().

**Comment by sn0wcat** [comment, marked as answer]
that is afaik not changeable.
